akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsMessages.scala

```diff
@@ -4,8 +4,10 @@
 
 package akka.cluster.metrics.sample
 
+import akka.serialization.jackson.CborSerializable
+
 //#messages
-final case class StatsJob(text: String)
-final case class StatsResult(meanWordLength: Double)
-final case class JobFailed(reason: String)
+final case class StatsJob(text: String) extends CborSerializable
+final case class StatsResult(meanWordLength: Double) extends CborSerializable
+final case class JobFailed(reason: String) extends CborSerializable
```

Comment:
Quite noisy with the marker everywhere. I guess for typed you'd mark the top message type rather though so not as bad?

Reply:
exactly